qutebrowser/config/configfiles.py

```diff
@@ -566,12 +566,18 @@ def _handle_error(self, action: str, name: str) -> typing.Iterator[None]:
 
     def finalize(self) -> None:
         """Do work which needs to be done after reading config.py."""
+        if config.instance.warn_autoconfig:
+            desc = configexc.ConfigErrorDesc("autoconfig loading not specified",
+                                             "Your config.py should call either `config.load_autoconfig()` (to load settings configured via the GUI) or `config.load_autoconfig(False)` (to not do so)")
+            self.errors.append(desc)
         self._config.update_mutables()
 
-    def load_autoconfig(self) -> None:
+    def load_autoconfig(self, load_config: bool = True) -> None:
         """Load the autoconfig.yml file which is used for :set/:bind/etc."""
+        config.instance.warn_autoconfig = False
         with self._handle_error('reading', 'autoconfig.yml'):
-            read_autoconfig()
+            if load_config:
```

Comment:
I'd probably put this around the `with ...` block, as no error can happen if we're doing nothing - but doesn't matter much, feel free to leave it as-is if you prefer.

Reply:
yeah, it's more logical to move that